### PR TITLE
add support for node 16

### PR DESCRIPTION
**Pull request by DemianD**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,25 +40,26 @@
...
+    "node-addon-api": "^3.2.1",
+    "@mapbox/node-pre-gyp": "^1.0.5"
   },
   "devDependencies": {
     "@mapbox/cloudfriend": "^1.9.0",
     "aws-sdk": "2.x",
     "eslint": "3.5.0",
-    "mocha": "^5.2.0"
+    "mocha": "^5.2.0",
+    "zeromq": "^5.2.8"
```
Comment (mscdex) [inline package.json:51]: This seems unnecessary?

File: scripts/build_against_node.sh [hunk 2]
```diff
@@ -28,6 +28,7 @@ export NODE_SQLITE3_JSON1=no
 if [[ $(uname -s) == 'Darwin' ]]; then
     brew update
     brew install sqlite
+    brew install zeromq
```
Comment (mscdex) [inline scripts/build_against_node.sh:31]: Ditto

**Comment by RocketFlashXD** [comment]
is sqlite3 ever gonna be supported on node v16?

**Review by UserJHansen — CHANGES_REQUESTED** [review]
Try deleting the package-lock and all references to zeromq, it looks like it is causing compile issues

**Comment by daniellockyer** [comment]
Hey @DemianD 👋🏻 Node 16 has been added to GitHub Actions CI and `sqlite3` should be able to compile on Node 16, or use the prebuilt binaries if they're available for your platform. 🙂 Given that, I'm going to close this PR 🙂

**Pull request closed without merging**